Bridge Hub preliminary docs (#5771)
w3f/polkadot-wiki: docs/learn/learn-guides-DOT-KSM-bridge.md

            "X2": [
              {
                "GlobalConsensus": "Kusama"
              },
              {
                "Parachain": "1,000"
              }
            ]
          }
        }
      },
      "beneficiary": {
        "V3": {
          "parents": "0",
          "interior": {
            "X1": {
              "AccountId32": {
                "network": null,
                "id": "0x9e4e7009937c56d267338762a60ed004293afd40e7c2081847c12cb63c76a818"
              }
            }
          }
        }
      },
      "assets": {
        "V3": [
          {
            "id": {
              "Concrete": {
                "parents": "1",
                "interior": "Here"
              }
            },
            "fun": {
              "Fungible": "30,000,000,000"
            }
          }
        ]
      },
      "fee_asset_item": "0",
      "weight_limit": "Unlimited"
    },
    "method": "limitedReserveTransferAssets",
    "section": "polkadotXcm"
  }
}
```

Once this [extrinsic](https://assethub-polkadot.subscan.io/extrinsic/6028374-2) is signed and
submitted, it is broadcast to Polkadot Asset Hub nodes. As this is a reserve asset transfer, the DOT

> is transferred to a dedicated account on Polkadot Asset Hub and

Reviewer: ```suggestion
is transferred to the destination's sovereign account on Polkadot Asset Hub and
```

[the wrapped DOT is issued](https://assethub-kusama.subscan.io/extrinsic/6758392-0?event=6758392-1)
as a foreign asset and deposited onto the destination account on Kusama Asset Hub. The foreign asset
balances of any account on Kusama Asset Hub can be queried on-chain through the
`foreignAssets`pallet as shown below.

![Wrapped DOT Balance](../assets/bridge-hub/KAH-DOT-Balance.png)

> ## Transfer KSM to Polkadot Asset Hub

Reviewer: This section is incomplete

Reviewer 2: Yeah, there is a bug in BagPipes that's messing with Kusama AssetHub's assets dropdown. Waiting for their response on the forum post. Worst case, I'll use PJS extrinsic from the tutorial you posted and push this PR today.

<img width="406" alt="image" src="https://github.com/w3f/polkadot-wiki/assets/86818441/bfa1c3b3-d559-417d-9e9b-d57d70ca37d7">
